src/dune_rules/melange/melange_stanzas.ml

```diff
@@ -92,7 +94,7 @@ module Emit = struct
                  ])
          in
          field "target" (plain_string (fun ~loc s -> of_string ~loc s))
-       and+ alias = field_o "alias" Alias.Name.decode
+       and+ alias_loc, alias = located (field_o "alias" Alias.Name.decode)
```

Comment:
```suggestion
       and+ alias_loc, alias = field_o "alias" (located Alias.Name.decode)
```

Reply:
If I do this change, then I have to `alias = Option.map ~f:snd alias` or similar when creating the record that will be returned from the function:

https://github.com/ocaml/dune/blob/4245029f2914bcfdca915e9916a5155a0afa95fb/src/dune_rules/melange/melange_stanzas.ml#L125

I'd rather keep it as is, but i don't have a strong opinion.

Reply:
no strong opinion